dydx3/starkex/starkex_resources/signature.py

```diff
@@ -118,6 +135,9 @@ def generate_k_rfc6979(msg_hash: int, priv_key: int, seed: Optional[int] = None)
 
 
 def sign(msg_hash: int, priv_key: int, seed: Optional[int] = None) -> ECSignature:
+    if CPP_LIB_PATH is not None:
+        return cpp_sign(msg_hash=msg_hash, priv_key=priv_key, seed=seed)
```

Comment:
It would be cleaner if we don't have to modify this file, which is Starkware's code. Can you add a file `starkex/crypto.py` that simply includes proxy functions that either call out to the python or cpp implementations?